shell/platform/fuchsia/flutter/flatland_connection.cc

```diff
@@ -13,22 +13,22 @@ namespace flutter_runner {
 
 namespace {
 
-fml::TimePoint GetNextPresentationTime(fml::TimePoint now,
-                                       fml::TimePoint next_presentation_time) {
-  return now > next_presentation_time
-             ? now + flutter_runner::kDefaultFlatlandPresentationInterval
-             : next_presentation_time;
+// Helper function for traces.
+double DeltaFromNowInNanoseconds(const fml::TimePoint& now,
+                                 const fml::TimePoint& time) {
+  return (time - now).ToNanoseconds();
 }
 
 }  // namespace
 
+// This function takes in all relevant information to determining when should
```

Comment:
Nit: Does this comment belong on the constructor or somewhere else?

Reply:
Ahh leftover comment from an old helper function that is removed now. Erased it.